deployment/packer/cac.json

```diff
@@ -0,0 +1,94 @@
+{
+  "variables": {
+    "local_project_directory": ".",
+    "aws_access_key": "",
```

Comment:
Could these be set from the `aws_access_key_id` and `aws_secret_access_key` vars set in the `group_vars` for `cac_secrets`?